kbase-extension/static/kbase/js/common/jobs.js

```diff
@@ -619,6 +738,12 @@ define(['common/errorDisplay', 'common/format', 'common/html', 'common/props', '
                 })
                 .join(', ');
 
+        if (statuses.retried) {
+            summaryHtml += ` (${
+                statuses.retried > 1 ? statuses.retried + ' jobs' : '1 job'
+            } retried)`;
+        }
```

Comment:
The status now reports the most current jobs, and adds in brackets the number of jobs that have been retried (not the number of retries).